lib/components/Checkbox.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { PropTypes } from 'react';
+import InputLabel from './InputLabel';
+// import './Checkbox.css';
+
+/**
+ * Checkbox with label
+ * 
+ * @param {string, required} id input's id and label's for attributes
+ * @param {string, required} label Text for the label
+ * @param {string} name defaults to 'checkboxs'. Sets name attribute
+ * @param {string} value Sets value attribute. If not provided the id is used.
+ * @param {bool} checked  If true, item is initially set to checked
+ * @param {bool} disabled If item, item is disabled
+ */
+export default function Checkbox({id, label, name, value, checked, disabled }) {
+  return (
+    <div>
+      <input
+        id={id}
```

Comment:
I agree it would be awesome not to have to specify IDs and `uniqueIdForComponent` is very clever! 

As to your second point about not needing the `id`s, are you 100% sure?  Everything I'm finding indicates that the `<label for="theId">` and `<input id="theId">` is required. See [here](http://www.section508.va.gov/support/html/10_2.asp) and [here](http://webaim.org/techniques/forms/controls) for example. Should we double check with Galina before making a call?